asking-questions/readme.md

```diff
@@ -0,0 +1,120 @@
+# Asking Questions 101
+
+## Prep
+
+- [ ] Read this article about [Mental Models](https://jamesclear.com/mental-models)
+- [ ] Load this webpage, which stores the problems we will work on today. [https://cyf-workshops.netlify.app/asking-questions](asking-questions)
```

Comment:
This is 404ing for me - just checking you're going to deploy this once this PR goes through

Reply:
https://deploy-preview-84--cyf-workshops.netlify.app/asking-questions/